### ISSUE TITLE
Incorrect a11y markup when first point is null

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#14349]
Example: https://jsfiddle.net/thomas_kennisnet/u3qw9fa2/3/show 

The marker group has `aria-hidden` while the graph group is not hidden as it should be.
